.travis.yml

```diff
@@ -16,8 +16,9 @@ install:
   - pip install --upgrade -r requirements-tests.txt
   - pip install --upgrade .
 script:
-  - pytest -vv --cov=vermouth vermouth --hypothesis-show-statistics
+  - coverage run $(which pytest) -vv vermouth --hypothesis-show-statistics
 after_success:
+  - coverage run --omit='*/bin/pytest'
```

Comment:
Why this change?

Reply:
Was meant to be `coverage report --omit='*/bin/pytest'``.

Reply:
The question was more general: Why invoke `coverage` instead of `pytest`?

Reply:
The detail is in the commit message, but the idea is that calling python through subprocess in the tests breaks how coverage reads its config file. This however, seems to occur only because pytest-cov tries to do some magic to get the coverage of the code we run through subprocess. Invoking coverage fixes the problem.

Reply:
https://coverage.readthedocs.io/en/coverage-4.2/subprocess.html